config/locales/en.yml

```diff
@@ -3729,6 +3736,27 @@ en:
       new:
         heading: "New attribute"
         description: "Changes to this project attribute will be reflected in all projects where it is enabled. Required attributes cannot be disabled on a per-project basis."
+    project_life_cycle_step_definitions:
+      heading: "Project lifecycle"
+      heading_description: "Project lifecycle define the project stages and gates that can be used for your project planning and will appear in the overview page of each project. These attributes can be enabled or disabled but not re-ordered at a project level."
```

Comment:
There is a grammar error in here. It would have to be "Project lifecycle defines" (with the s). 

In any case, I suggest to rephrase the text a bit:

"The project lifecycle consists of stages and gates that structure your projects. They will appear in the overview page of a project and be available in the projects list. Individual stages and gates can be selected to customize a project but reordering is not possible on a project level."

Reply:
@marcalcobe

Reply:
Okay for me.